.github/workflows/arm64.yml

```diff
@@ -83,6 +83,21 @@ jobs:
             --ignore=tests/scenarios/test_sync.py \
             --ignore=tests/scenarios/test_transformation.py \
             -k "not test_search"
+      - name: "Persist test failure flag"
+        run: echo "TEST_FAILURE=true" >> $GITHUB_ENV
+        if: failure()
+      - uses: actions/checkout@v3
+        if: always()
+        with:
+          persist-credentials: false
```

Comment:
what's the purpose of this?

Reply:
See https://github.com/Alfresco/acs-packaging/pull/2449#discussion_r1173432354
I also disabled option that the auth token is persisted in the local git config.

Reply:
I didn't follow everything here but consider having the main job that set as outputs whatever is needed to handle the jira creation and then have a dedicated job that start only if previous one failed, it could help in keeping things more separated

Reply:
The jira creation has been moved to the dedicated job. 
https://alfresco.atlassian.net/browse/ACS-5125 - this is an example of created/closed Jira issue after the change.